harvard-lts/CURIOSity
# Rails bump

**Bump Rails to 6.1.7.1**
* * *


# What does this Pull Request do?
Upgrades Rails and adds the seeds.rb to docker ignore since it will never be used.

# How should this be tested?

Spin up this branch locally and make sure things still work (things come up, you can search, you can log in, you can do a harvest and make an exhibit).

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
